win-odbc64/odbcclient/drvr35/lob.cpp

```diff
@@ -54,8 +54,12 @@ namespace ODBC {
         CStmt *pStatement = (CStmt *)hstmt;
         IDL_long extractLen = length;
 
+        IDL_string lobHandleTemp = (IDL_string)malloc(lobHandleLen + 1);

```

Comment:
Is there a place where this string will be deleted? If not, is this a memory leak?